### PR TITLE
Fixes Code style for non-public, non-static fields #8387

### PR DESCRIPTION
**Pull request by Arnold2381**
## Purpose / Description
Change the formatting issues according to #8370 

## Fixes
Fixes some of #8387 

## Approach
Followed #8370, added those changes ran gradlew lintRelease to generate a report of all the issues and solve errors accordingly. 

## Checklist
_Please, go through these checks before submitting the PR._

- [x] You have not changed whitespace unnecessarily (it makes diffs hard to read)
- [x] You have a descriptive commit message with a short title (first line, max 50 chars).
- [x] Your code follows the style of the project (e.g. never omit braces in `if` statements) 
- [x] You have commented on your code, particularly in hard-to-understand areas
- [x] You have performed a self-review of your own code
- [x] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [x] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

**Comment by Arnold2381** [comment]
@david-allison-1 I think you can merge #8370 with master and run the tests again after this will be merged. we are good to go.

In 5 files I added suppress warning just check those once too.

File: AnkiDroid/src/main/java/com/ichi2/anki/dialogs/ExportDialog.java [hunk 1]
```diff
@@ -11,6 +12,7 @@
 import com.ichi2.anki.analytics.AnalyticsDialogFragment;
 import com.ichi2.utils.BundleUtils;
 
+@SuppressLint("FieldNamingPatternDetector")
```
Comment (david-allison-1) [inline AnkiDroid/src/main/java/com/ichi2/anki/dialogs/ExportDialog.java:15]: This one should be fine to translate

File: AnkiDroid/src/main/java/com/ichi2/anki/multimediacard/activity/MultimediaEditFieldActivity.java [hunk 2]
```diff
@@ -343,6 +343,7 @@ private void recreateEditingUIUsingCachedRequest() {
     }
 
     public void onRequestPermissionsResult (int requestCode, @NonNull String[] permissions, @NonNull int[] grantResults) {
+        super.onRequestPermissionsResult(requestCode, permissions, grantResults);
```
Comment (david-allison-1) [inline AnkiDroid/src/main/java/com/ichi2/anki/multimediacard/activity/MultimediaEditFieldActivity.java:346]: Please revert/rebase this out - we shouldn't bundle in changes into a refactoring PR

File: AnkiDroid/src/main/java/com/ichi2/anki/multimediacard/activity/TranslationActivity.java [hunk 3]
```diff
@@ -63,6 +64,7 @@
  * Activity used now with Glosbe.com to enable translation of words.
  * FIXME why isn't this extending from our base classes?
  */
+@SuppressLint("FieldNamingPatternDetector")
```
Comment (david-allison-1) [inline AnkiDroid/src/main/java/com/ichi2/anki/multimediacard/activity/TranslationActivity.java:67]: Same - I think this should be fine to translate?

File: AnkiDroid/src/main/java/com/ichi2/libanki/AnkiPackageExporter.java [hunk 4]
```diff
@@ -150,7 +151,7 @@ protected String stripHTML(@NonNull String text) {
     }
 }
 
-
+@SuppressLint("FieldNamingPatternDetector")
```
Comment (david-allison-1) [inline AnkiDroid/src/main/java/com/ichi2/libanki/AnkiPackageExporter.java:154]: Just do it for the individual field, not the class, you can do this with the Alt+Enter menu on the error

File: AnkiDroid/src/main/java/com/ichi2/libanki/AnkiPackageExporter.java [hunk 5]
```diff
@@ -566,6 +568,7 @@ private void _addDummyCollection(ZipFile zip, Context context) throws IOExceptio
  * 
  * @author Tim
  */
+@SuppressLint("FieldNamingPatternDetector")
```
Comment (david-allison-1) [inline AnkiDroid/src/main/java/com/ichi2/libanki/AnkiPackageExporter.java:571]: I don't see why this is here

File: AnkiDroid/src/main/java/com/ichi2/libanki/Sound.java [hunk 6]
```diff
@@ -394,6 +395,7 @@ public void notifyConfigurationChanged(VideoView videoView) {
     }
 
     /** #5414 - Ensures playing a single sound performs cleanup */
+    @SuppressLint("FieldNamingPatternDetector")
```
Comment (david-allison-1) [inline AnkiDroid/src/main/java/com/ichi2/libanki/Sound.java:398]: ditto

File: AnkiDroid/src/main/java/com/ichi2/libanki/sync/UnifiedTrustManager.java [hunk 7]
```diff
@@ -34,6 +36,7 @@
 // We try the local manager first.
 // Cached the accepted issuers.
 // Did not ignore NoSuchAlgorithmException
+@SuppressLint("FieldNamingPatternDetector")
```
Comment (david-allison-1) [inline AnkiDroid/src/main/java/com/ichi2/libanki/sync/UnifiedTrustManager.java:39]: ditto

**Review by david-allison-1 — CHANGES_REQUESTED** [review]
Thanks!!

File: AnkiDroid/src/main/java/com/ichi2/anki/dialogs/ExportDialog.java [hunk 8]
```diff
@@ -21,8 +19,8 @@
         void dismissAllDialogFragments();
     }
 
-    private final int INCLUDE_SCHED = 0;
-    private final int INCLUDE_MEDIA = 1;
+    private final int mINCLUDESCHED = 0;
+    private final int mINCLUDEMEDIA = 1;
```
Comment (david-allison-1) [inline AnkiDroid/src/main/java/com/ichi2/anki/dialogs/ExportDialog.java:23]:
```java
    private final static int INCLUDE_SCHED = 0;
    private final static int INCLUDE_MEDIA = 1;
```

File: AnkiDroid/src/main/java/com/ichi2/libanki/AnkiPackageExporter.java [hunk 9]
```diff
@@ -568,9 +568,8 @@ private void _addDummyCollection(ZipFile zip, Context context) throws IOExceptio
  * 
  * @author Tim
  */
-@SuppressLint("FieldNamingPatternDetector")
 class ZipFile {
-    private final int BUFFER_SIZE = 1024;
+    private final int mBUFFERSIZE = 1024;
```
Comment (david-allison-1) [inline AnkiDroid/src/main/java/com/ichi2/libanki/AnkiPackageExporter.java:572]:
```java
    private static final int BUFFER_SIZE = 1024;
```

**Review by david-allison-1 — CHANGES_REQUESTED** [review]
A few of these should have been constant

Other than that, good to go

**Comment by david-allison-1** [comment]
I missed this! This is awesome, thank you so much!!!!!!!!

**Comment by Arnold2381** [comment]
> > @david-allison-1 I think you can merge #8370 with master and run the tests again after this will be merged. we are good to go.
> > In 5 files I added suppress warning just check those once too.
> > [image](https://user-images.githubusercontent.com/46752548/114226187-e61e0700-9990-11eb-932a-c7566a7dc191.png)
> > Got this finally haha.
> 
> I missed this! This is awesome, thank you so much!!!!!!!!

No worries btw i will take some more time to resolve all this, just trying to rebase since then hope I get it.

File: AnkiDroid/src/main/java/com/ichi2/anki/multimediacard/activity/MultimediaEditFieldActivity.java [hunk 10]
```diff
@@ -342,6 +343,7 @@ private void recreateEditingUIUsingCachedRequest() {
         recreateEditingUi(mCurrentChangeRequest);
     }
 
+    @SuppressLint("MissingSuperCall")
```
Comment (david-allison-1) [inline AnkiDroid/src/main/java/com/ichi2/anki/multimediacard/activity/MultimediaEditFieldActivity.java:346]: Just leave it for another pull request

**Comment by Arnold2381** [comment]
@david-allison-1 I tried my best was successful the first time, and then for the second one too I did rebase and edited in that only but I don't know why it got pushed as a seperate commit.

But now I got most of it how is it done next time would be perfect.

**Review by david-allison-1 — CHANGES_REQUESTED** [review]
Just one nit, then we're good to go! Thanks so much for all the effort

File: AnkiDroid/src/main/java/com/ichi2/anki/multimediacard/activity/MultimediaEditFieldActivity.java (hunk 10 above)
Comment (Arnold2381) [reply AnkiDroid/src/main/java/com/ichi2/anki/multimediacard/activity/MultimediaEditFieldActivity.java:346]: So should i remove this ?

**Comment by Arnold2381** [comment]
@david-allison-1  is everything okay now?

**Review by david-allison-1 — APPROVED** [review]
Fantastic!!

**Comment by Arnold2381** [comment]
Thank you!!
Finally, I learnt git rebase!! 
Relief :)

**Review by mikehardy — APPROVED** [review]
These kinds of changes - cleaning codebase - are probably some of the more brutal changes to do as a developer because they seem to just go on forever like a bowl of soup that is more soup than you want ;-). But seriously they help make this project sustainable for the next decade - thank you very much for plowing through the whole codebase and getting this done, I appreciate it

**Pull request merged**